### DISCUSSION TITLE
Using prefixes with config files

### DISCUSSION THREAD

**Discussion by ceng-p** [spf13/viper#1591] [Q&A]
Hello! 

In the "Working with environment variables" section there is this example.
```
SetEnvPrefix("spf") // will be uppercased automatically
BindEnv("id")

os.Setenv("SPF_ID", "13") // typically done outside of the app

id := Get("id") // 13
```

If I want to work with a config file, let's say `dev.env` that contains variables that have the same prefix "spf". Is there a way to get those config variables without specifying "SPF" in the call to `Get` ?

For example with this file
``` 
spf_user_id=abc
```
I would have to call `userId := Get("spf_user_id")` instead of just calling `Get(user_id)`

**Comment by sagikazarmark** [comment]
Not at the moment. You could try using `RegisterAlias`, but I'm not sure if it's going to work in this case.

A workaround I'd try is loading env vars from the file using some dotenv tooling.
